site/e2e/README.md

```diff
@@ -20,3 +20,26 @@ pnpm playwright:test
 # Run a specific test (`-g` stands for grep. It accepts regex).
 pnpm playwright:test -g '<your test here>'
 ```
+
+# Using nix
```

Comment:
I think it's probably worth linking to https://search.nixos.org/packages?channel=unstable&type=packages&query=playwright-driver somewhere in this section as a quick way to figure out what the "right" Playwright version is

Reply:
:+1: